Gemfile

```diff
@@ -14,6 +14,7 @@ group(:development, :test) do
   gem "simplecov"
   gem 'rack', "~> 1.5.1"
   gem 'cheffish', "~> 1.3"
+  gem 'timecop'
```

Comment:
I'm fairly against adding timecop as a dependency just for testing. I think we can stub the time easily just using RSpec.